distributed/src/main/java/com/orientechnologies/orient/server/hazelcast/OHazelcastPlugin.java

```diff
@@ -844,7 +845,28 @@ protected HazelcastInstance configureHazelcast() throws FileNotFoundException {
         .setMergePolicy(OHazelcastMergeStrategy.class.getName());
     // Disabled the shudown hook of hazelcast, shutdown is managed by orient hook
     hazelcastConfig.setProperty("hazelcast.shutdownhook.enabled", "false");
-
+    boolean enableKubernetesDiscoveryEnvVar = false;
+    String enableKubernetesDiscoveryEnvVarStr = System.getenv("ENABLE_KUBERNETES_DISCOVERY");
```

Comment:
Hi,

Is this really  needed ? as far as I understood this is just overriding the settings in the hazelcast.xml, I see two possible alternative solution to this, one is this: https://hazelcast.com/blog/xml-variables/ (if this version of HZ support it), and another is use OGlobalCofiguration for the variable, that's where all the variable a configured in OrientDB

Reply:
well, we need a way to "configure" the dockerized orientDB to use kubernetes discovery. This works. The other two options might be more consistent with the rest of our configs. I'll check.